benchmarks/torchbench_model.py

```diff
@@ -218,16 +230,29 @@ def is_compatible(self,
                     dummy_benchmark_model,
                     benchmark_experiment,
                     use_strict_deny=False):
+    name = dummy_benchmark_model.model_name
     deny_list = STRICT_DENY_LIST if use_strict_deny else DENY_LIST
-    if dummy_benchmark_model.model_name in deny_list:
-      for deny_experiment_config in deny_list[dummy_benchmark_model.model_name]:
-        matched = True
-        for k, v in deny_experiment_config.items():
-          if getattr(benchmark_experiment, k) != v:
-            matched = False
-            break
-        if matched:
-          return False
+
+    if name in self.skip["skip"]:
+      return False
+
+    if name in self.skip["test"].get(benchmark_experiment.test, {}):
+      return False
+
+    if name in self.skip["device"].get(benchmark_experiment.accelerator, {}):
+      return False
+
+    if name in self.skip["multiprocess"]:
+      # No support for multiprocess, yet. So, skip all benchmarks that
+      # only work with it.
+      return False
```

Comment:
I wonder if we should include an "INFO" message when skipping models so it's clear on the logs - unless it gets reported elsewhere. wdyt @zpcore?

Reply:
The scripts already report when a model is skipped. However, the reason is not logged. Do you think we should log that, too?